### PR TITLE
Develop rm collectionreporting ar

### PR DESCRIPTION
**Pull request by anthonyr-ten24**

### CODE REVIEW

File: org/Hibachi/client/src/collection/components/swdisplayitemaggregatesimple.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            link: function(scope, element,attrs,displayOptionsController){
+                scope.showDisplayItem = false;
+                scope.aggegate ={};
+                scope.aggegate.selectedAggregate = '';
+                
+                
+                scope.aggregateOptions = [
+                    {id:'average', value:'Average'},
+                    {id:'count', value:'Count'},
+                    {id:'sum', value:'Sum'}
```
Comment (migueltarga) [inline org/Hibachi/client/src/collection/components/swdisplayitemaggregatesimple.ts]: lets add MIN and Max to this list.

File: org/Hibachi/client/src/collection/components/swdisplayoptions.ts [hunk 2]
```diff
@@ -123,6 +124,12 @@ class SWDisplayOptions{
                     return title;
                 };
 
+                scope.displayAggregates = function(){
+                    
+                    scope.showAggregates = (scope.showAggregates == true ? false : true)
```
Comment (migueltarga) [inline org/Hibachi/client/src/collection/components/swdisplayoptions.ts]: you can simplify:  `scope.showAggregates = !scope.showAggregates`

File: org/Hibachi/client/src/collection/components/swdisplayoptions.ts [hunk 3]
```diff
@@ -123,6 +124,12 @@ class SWDisplayOptions{
...
+                    console.log("Changed agg:",scope.showAggregates);
```
Comment (migueltarga) [inline org/Hibachi/client/src/collection/components/swdisplayoptions.ts]: remove debug

**Comment by rmarchandTen24** [comment]
@anthonyr-ten24 we should extend the existing directive so we don't have to maintain two seperate directives for a small change

**Pull request closed without merging**